stylesheets/style.css

```diff
@@ -5,3 +5,342 @@ main-titles dark-grey: #2C303F;
+  width: 50%;
+  margin: 50px;
+}
+main div button.white-btn{
+  background:white;
+  border:1px solid #192592;
+  color: #192592;
+}
+footer{
+  width: 100%;
+  position: relative;
+  height: auto;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  flex-direction: column;
+  background: white;
+  margin: 30px 0 0 0;
+ box-shadow: 0px -23px 8px -29px #5b5e6d;
+ font-size: 0.8rem;
+}
+footer div{
+  width: 100%;
+  position: relative;
+  height: auto;
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+footer div:nth-child(2){
+  background:  #F4F3F4;
+  margin-top: 40px;
+}
+footer div ul{
+  width: 40%;
+  padding-left: 5%;
+}
+footer ul li{
+  margin: 20px 0;
+}
+footer div:first-child ul li:first-child{
+  font-weight: 800;
+}
+footer div:first-child img{
+  margin-right: 70%;
+  width: 30%;
+}
+footer div:nth-child(2) ul{
+  width: 100%;
+  margin: 15px 0 10px 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: row;
+  padding: 0;
+}
+footer div:nth-child(2) ul li{
+ display: inline;
+ margin: 15px;
+ text-align: center;
+}
+footer div:nth-child(2) div{
+  margin-top: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: row;
+  margin-bottom: 15px;
+}
+
+footer div:nth-child(2) div img{
+  width: 30px;
+  margin: 10px 20px 10px 10px;
+}
+footer div:nth-child(2) div i{
+  font-size: 20px;
+  color: #5b5e6d;
+  margin: 0 10px;
+}
+footer div:nth-child(2) div i.fa-chevron-down{
+  font-size: 20px;
+}
+
+.text-center{
+  text-align: center;
+}
+.margin-small{
+  margin: 30px 0px 0px 0px;
+}
+@media screen and (min-device-width: 767px){ 
+  header {
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+input {
+  width: 50%;
+  margin: 20px 20px 20px 0px;
+}
+header button, main div button {
+  width: 30%;
+  margin: 20px 0px;
+  padding: 20px 10px;
+}
+main div {
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+main div button.white-btn{
+  width: 60%;
+}
+main div img {
+  width: 35%;
+  margin: 40px;
+}
+footer div:first-child {
+  flex-wrap: nowrap;
+}
+footer div:first-child img {
+  width: 15%;
+  margin-right: 0;
+}
+footer div:first-child ul{
+  width: 19%;
+  padding-left: 2%;
+}
+footer div:nth-child(2) {
+  flex-wrap: nowrap;
+  align-items: center;
+}
+footer div:nth-child(2) div {
+  margin: 0;
+}}
+@media screen and (min-device-width: 961px) and (max-device-width: 1024px) { 
+  h2 {
+    margin: 50px 0 0;
+}
+  main div img {
+    width: 25%;
+    margin: 40px;
+  }
+  main div button.white-btn {
+    width: 40%;
+    margin: 20px 30%;
+}
+}
+@media screen and (min-device-width: 1024px) { 
+  h2 {
+    margin: 50px 0 0;
+}
+  main div img {
+    width: 10%;
+    margin: 40px;
+  }
+  main div button.white-btn {
+    width: 40%;
+    margin: 20px 30%;
+}
+  nav ul li{
+    display: inline;
+    margin: 0px 20px;
+  }
+  nav ul .fas{
+    display: none;
+  }
+}
```

Comment:
Really great job with the css here, the only feedback I would say is maybe add some comments so that it is clear what part of your css manages what.